README.md

```diff
@@ -21,32 +39,33 @@ $ psql -f db/schema.sql -d soapbox_dev
 
 2. **Build and install the Soapbox API server (soapboxd) and CLI client (soapboxcli):**
 ``` shell
+$ mkdir -p $GOPATH/src/github.com/adhocteam
+$ ln -s /path/to/soapbox $GOPATH/src/github.com/adhocteam/soapbox
+$ cd $GOPATH/src/github.com/adhocteam/soapbox
+$ go get -u github.com/golang/dep/cmd/dep
+$ dep ensure
 $ go get -d -v ./...
 $ go install ./...
 ```
 
-3. **Install Terraform**
-
-Download and install Terraform from [here](https://www.terraform.io/downloads.html).
-
-4. **Run the API server:**
+3. **Run the API server:**
 ``` shell
-$ PGDATABASE=soapbox_dev PGSSLMODE=disable soapboxd &
+$ PGDATABASE=soapbox_dev PGPASSWORD=yourpgpass PGSSLMODE=disable soapboxd &
```

Comment:
I would not add PGPASSWORD here as a recommendation, because everyone's db environment (based on their pg_hba.conf file) will be different with respect to auth.